Dockerfile

```diff
@@ -1,21 +1,13 @@
-FROM openjdk:17-slim
+FROM arm64v8/eclipse-temurin:17
 
-MAINTAINER Benjamin Rögner "benjamin.roegner@here.com"
-MAINTAINER Lucas Ceni "lucas.ceni@here.com"
-MAINTAINER Dimitar Goshev "dimitar.goshev@here.com"
+ARG NAKSHA_VER
```

Comment:
If the gradle build generates X version of jar, somehow, I don't see a practical need for someone using this docker and pass some Y version of Jar to build an image. We can rather, directly pick up the `build/libs/naksha-*-all.jar` (if that is possible?) ... unless there was some issue observed while doing so.

Reply:
I was thinking of a scenario when one has multiple fatjars in libs directory - then this would potentially be a failsafe mechanism for choosing single jar.
If we're fine with the assumption of single file residing there, I can omit this :)

Reply:
Today, there is no scenario like that. At the root level, there is only one fat jar.
In future, if we have multiple fat jars, then likely we will need separate docker picking up individual fat jar. So, 1 docker per jar, so there is no conflict of env / settings by bundling all in the same image. But we can still assume that there will be same version as an outcome of a build process ... but multiple fat-jar is potential future :) so, can start with simple